README.md

```diff
@@ -490,7 +490,7 @@ uintBuf := fasthttp.AppendUint(nil, 1234)
     go middleware web framework which is based on fasthttp.
   * [websocket](https://github.com/leavengood/websocket) - Gorilla-based
     websocket implementation for fasthttp.
-
```

Comment:
Please add this space back and then we can merge it (keep the formatting in the whole document consistent with 2 spaces before a # head).